examples/all-clusters-app/all-clusters-common/src/laundry-washer-controls-delegate-impl.cpp

```diff
@@ -0,0 +1,42 @@
+#include <app/clusters/laundry-washer-controls-server/laundry-washer-controls-server.h>
+#include <app/util/config.h>
+#include <laundry-washer-controls-delegate-impl.h>
+
+using namespace std;
```

Comment:
where is std used?

In other files we generally do like you did in the header:

```
namespace chip {
namespace app {
namespace Clusters {
namespace LaundryWasherControls {
...
```

any reason for the many using here instead?

Reply:
They are the ones I forget removing. Will remove them.